sdk/identity/azure-identity/azure/identity/aio/_credentials/chained.py

```diff
@@ -22,6 +23,15 @@ class ChainedTokenCredential(SyncChainedTokenCredential):
     :type credentials: :class:`azure.core.credentials.TokenCredential`
     """
 
+    async def __aenter__(self):
+        pass
+
+    async def close(self):
+        """Close the transport sessions of all credentials in the chain."""
+
+        for credential in self.credentials:
+            await credential.close()
```

Comment:
Is there a reason that we are serializing the `close` calls? It looks odd. I would have expected us to do them concurrently rather than `await`ing them one by one...

Reply:
Some combination of inattention and haste. Thanks for catching this. I'll `gather` them.